opentelemetry/sidecar/otelcol.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  name: otel-collector-example
+---
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
The SA might not be needed as the sidecar will use the app's SA.

Then we should document that the openshift resources will not work OOTB and the app's SA needs to be changed.